BALSAMIC/commands/options.py

```diff
@@ -370,6 +370,20 @@
     help="Sample configuration file",
 )
 
+OPTION_SENTIEON_INSTALL_DIR = click.option(
+    "--sentieon-install-dir",
+    type=click.Path(exists=True, resolve_path=True),
+    required=True,
+    help="Path to Sentieon install directory",
+)
+
+OPTION_SENTIEON_LICENSE = click.option(
+    "--sentieon-license",
+    required=True,
+    type=click.STRING,
+    help="Sentieon license in format IP:Port",
+)
```

Comment:
Maybe we could add a more explanatory names for these options:

```suggestion
OPTION_SENTIEON_INSTALL_DIR = click.option(
    "--sentieon-license-path",
    type=click.Path(exists=True, resolve_path=True),
    required=True,
    help="Path to Sentieon install directory",
)

OPTION_SENTIEON_LICENSE = click.option(
    "--sentieon-license-server",
    required=True,
    type=click.STRING,
    help="Sentieon license in format IP:Port",
)
```

Reply:
I agree with the second change of sentieon-license to sentieon-license-server but the sentieon-install-dir I think fits the way we name this variable elsewhere, and I think it's more correct since it's not a license-path but the path to where Sentieon is installed. I could see an argument for --sentieon-install-path but I think the "-dir" part is more in line with how we generally name these arguments in balsamic, with for instance --analysis-dir

Reply:
There's also however so many places in the code where sentieon-license would need to be renamed to sentieon-license-server 😓 I don't know if it's worth it

Reply:
we're also calling it sentieon_license in the pytests

Reply:
👍  I don't have a strong opinion on this, it's up to you

Reply:
But I think we'll have to do some refactoring in any case, It would be great to use these options directly from the config and not redefining them

Reply:
I have done the refactoring now, replacing the sentieon_install_dir, sentieon_exec, sentieon_license, sentieon_tnscope_dir, sentieon_dnascope_dir from the config to being retreived from a sentieon model ⭐ good suggestion!

Reply:
though I didn't change the sentieon_license name, my find and replace in pycharm is being very spotty so it doesn't find all cases of the variables when I search, it doesn't feel worth the time to change. But it still feels like a significant improvement regardless!

Reply:
fair enough, I think this is quiet an improvement already from what we had 💯